src/NHSD.BuyingCatalogue.Identity.Api/appsettings.json

```diff
@@ -7,13 +7,13 @@
   },
   "clients": [
     {
-      "clientId": "SampleClient",
-      "clientName": "Sample Client",
+      "clientId": "NodeClient",
+      "clientName": "Node Client",
       "allowedGrantTypes": "Code",
       "accessTokenLifetime": 3600,
       "allowOfflineAccess": false,
       "requireClientSecret": true,
-      "secret": "SampleClientSecret",
+      "secret": "NodeClientSecret",
```

Comment:
Unfortunately the FE have not changed the secret to match.

Reply:
I'll revert it.